style.css

```diff
@@ -0,0 +1,64 @@
+body {
+  margin: 0;
+  padding: 25px;
+  background-attachment: fixed;
+  background-size: 100% 100%;
+}
+
+.main-class {
+  background-image: url('image.svg');
```

Comment:
Great UI @Shedrack-Sunday 🤝

- [ ] [Optional] Kindly make sure your design is a close replica of the Figma design. Please add this icon I have provided in the screenshot below.
![button](https://user-images.githubusercontent.com/63917725/185646943-b84f3327-ddd4-441b-9faa-65339b100694.PNG)